### DISCUSSION TITLE
Multiple htmls entries

### DISCUSSION THREAD

**Discussion by NorbertBodziony** [crxjs/chrome-extension-tools#233] [Q&A]
Hey I am looking for a way to have multiple html entries. Not just popup.html.
![image](https://user-images.githubusercontent.com/38857779/157674477-dccf859d-3186-4047-b873-a0d55fa34893.png)
Something like this but if html is not in manifest its not added to build.
Thanks for help.

**Comment by NorbertBodziony** [comment]
![image](https://user-images.githubusercontent.com/38857779/157678542-0991cc83-d075-4c2c-af4e-db52caaa4168.png)
This works but break HMR.

**Reply by jacksteamdev** [reply]
Can you create an issue with details on how it breaks HMR? I want it to work this way, but I haven't had time to look into it fully.

Similar to #227 in that devtools extensions use HTML pages not declared in the manifest.

**Reply by NorbertBodziony** [reply]
So i found problem popup2.html and default popup.html referenced same tsx code that caused HMR to just close extension rather than refreshing. After i used different tsx code HMR works as intended.
IDK if its worth creating issue.
